### PR TITLE
Fix NavBar item to Community Configurations

### PR DESCRIPTION
**Pull request by sladyn98**

### CODE REVIEW

File: frontend/src/components/layouts/Header.js [hunk 1]
```diff
@@ -22,12 +22,12 @@ export const Header = () => {
 					</Link>
 					<Link to="/communityConfiguration">
 						<li className="nav-item">
-							<a className="nav-link" >Package Generation</a>
+							<a className="nav-link" >Community Configuration</a>
```
Comment (martinda) [inline frontend/src/components/layouts/Header.js:25]: Plural form would be better. We are presenting more than one to the user.

Comment (sladyn98) [reply frontend/src/components/layouts/Header.js:25]: Done

**Comment by martinda** [comment]
The tests appear to fail during Windows testing. Different OSes have different line terminations (to the despair of programmers). Check your code for how the comparison is done and how the files are loaded. I see a call to `Files.readAllBytes()` which may not be the best way to read a file due to the line ending characters (EOL). You should be able to find a way to load files without having to worry about line endings.

**Comment by sladyn98** [comment]
@martinda I searched on Google but cannot seem to find any other method to read. Is there something I am missing.should I just include or exclude new lines from the test files .

**Comment by sladyn98** [comment]
@martinda should I close and reopen that will kickstart the build again maybe it will pass since it was passing before

**Comment by martinda** [comment]
I think there is an issue with calling `Files.getAllBytes()`. I think it returns a different EOL depending on the OS. Maybe it does not matter in this case? You can try to re-trigger the test of course.

**Comment by sladyn98** [comment]
@martinda yeah I guess it does not matter since the previous builds were passing let's retrigger and check

**Comment by kwhetstone** [comment]
I had thought that the tests at one point were only supposed to be run on Linux?

**Pull request merged**